src/main/webapp/i18n/de/programmingExercise.json

```diff
@@ -32,6 +32,10 @@
                 "title": "Sequentielle Testläufe",
                 "description": "Aktiviere diese Checkbox um zuerst Struktur- und erst im Anschluss Verhaltenstests duchzuführen. Dieses Feature hilft Studenten dabei, sich auf die aktuell anliegende Herausforderung zu konzentrieren. Bei Aktivierung werden im Test Repository zusätzliche Informationen bereitgestellt. ACHTUNG: Kann nicht nachträglich geändert werden!"
             },
+            "automaticSubmissionRunAfterDueDate": {
+                "title": "Automatische Bewertung aller Ergebnisse nach dem Abgabedatum",
+                "description": "Aktiviere diese Option um eine automatische Bewertung aller Abgaben 1 Stunde nach dem Abgabedatum durchzuführen. Die hierdurch generierten Ergebnisse werden als 'bewertet' gekennzeichnet obwohl das Abgabedatum abgelaufen ist."
```

Comment:
```suggestion
                "description": "Aktiviere diese Option um eine automatische Bewertung aller Abgaben eine Stunde nach dem Abgabedatum durchzuführen. Die hierdurch generierten Ergebnisse werden als 'bewertet' gekennzeichnet obwohl das Abgabedatum abgelaufen ist."
```

Reply:
This is obsolete as I'll change the way the feature works (not limited to setting it 1 hour after the due date).